src/index.ts

```diff
@@ -24,6 +24,13 @@ program
     .option('-f, --folder-only <folder-name>', 'Folder Name for checkout')
     .action(checkout);
 
+program
+    .command('remove [name]')
+    .description('Remove a folder or NX project to the monorepo')
+    .option('-f, --folder-only', 'Remove only the folder without computing dependencies')
```

Comment:
Pls add screenshot for folder removal as well

Reply:
Screenshot for folder removal

![image](https://github.com/user-attachments/assets/14f01cb5-69fe-4419-9873-e975a9700fcf)